source/extensions/transport_sockets/tcp_stats/tcp_stats.cc

```diff
@@ -133,6 +133,13 @@ void TcpStatsSocket::recordStats() {
     config_->stats_.cx_tx_percent_retransmitted_segments_.recordValue(percent_retransmissions);
   }
 
+  const uint64_t mss = (tcp_info->tcpi_snd_mss > 0) ? tcp_info->tcpi_snd_mss : 1460;
+  // Convert packets to bytes.
+  auto snd_cwnd = tcp_info->tcpi_snd_cwnd * mss;
+  // Convert delivery & pacing rate to MB.
+  auto delivery_rate_mb = std::round(static_cast<float>(tcp_info->tcpi_delivery_rate) / 1000000);
```

Comment:
Don't covert these; there's better data available if the value is in bytes not megabytes. Same for the next one.